### ISSUE TITLE
Failure to install pyqt-qt4 dependency of qgis2

### ISSUE THREAD

**Issue by almcnall** [OSGeo/homebrew-osgeo4mac#198]
The error message following `brew install pyqt-qt4` is
```
==> Installing pyqt-qt4 from osgeo/osgeo4mac
==> Downloading https://downloads.sf.net/project/pyqt/PyQt4/PyQt-4.11.4/PyQt-mac-gpl-4.11.4.tar.gz
Already downloaded: /Users/almcnall/Library/Caches/Homebrew/pyqt-qt4-4.11.4.tar.gz
==> python configure.py --confirm-license --bindir=/usr/local/Cellar/pyqt-qt4/4.11.4/bin --destdir=/usr/local/Cellar/pyqt-qt4/4.11.4/lib/qt-4/python2.7/site-packages --sipdir=/usr/local
Error: ["pyqtconfig.py", ["expected replacement of #<Pathname:/usr/local/opt/qt-4> with #<Pathname:/usr/local/opt/qt-4>"]]
```

[Output of `brew gist-logs pyqt-qt4`](https://gist.github.com/83de8ed9e1125b1f3578164cc7094271
)

**Comment by dakcarto** [comment]
Hi @almcnall, thanks for reporting. Do `brew update` and try again. Unable to replicate here, but should be fixed with d22a424.

**Comment by itcarroll** [comment]
To see if I could replicate the error, I uninstalled my working pyqt-qt4 and on installing with your latest update saw:

```
Error: ["pyqtconfig.py", ["expected replacement of \"/usr/local/opt/qt-4\" with \"/usr/local/opt/qt-4\""]]
```

Clearly my `Formula["qt-4"].prefix` and `Formula["qt-4"].opt_prefix` are identical, and I confirmed there is no "/usr/local/opt/qt-4" anywhere in "pyqtconfig.py". Don't know how to debug ruby at all; is it expected for `inreplace` to fail when the string isn't found?

Not sure what could have changed since I installed qgis2 -- quite recently -- without seeing this until now.

**Comment by duanearnett** [comment]
@dakcarto Tried with that commit and I'm also receiving the same error that is being reported.

**Comment by mhkeller** [comment]
I'm getting the same error:
```
Error: ["pyqtconfig.py", ["expected replacement of \"/usr/local/opt/qt-4\" with \"/usr/local/opt/qt-4\""]]
```
I'm installing on a new installation of macOs 10.12.2 after running `brew update`

**Comment by itcarroll** [comment]
I meant to add earlier for anyone hitting this, if you `brew install --debug pyqt-qt4`, you can choose to ignore the error and things seem to proceed swimmingly.

**Comment by mhkeller** [comment]
@itcarroll thanks that worked for me.

**Comment by dakcarto** [comment]
Hi, finally able to replicate. Should be fixed in 44f1d0d. Please `brew update` and retry.

**Comment by almcnall** [comment]
success! Thanks.
